src/EditorFeatures/CSharpTest/InvertLogical/InvertLogicalTests.cs

```diff
@@ -594,6 +594,66 @@ void M(bool x, int a, object b)
 }", parseOptions: CSharp9);
         }
 
+        [Fact, WorkItem(64558, "https://github.com/dotnet/roslyn/issues/64558")]
+        public async Task InvertNumericIsGreaterThanPattern1_CSharp9()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M(bool x, int a, object b)
+    {
+        var c = a > 10 [||]&& a is > 20;
```

Comment:
just add test that shows the behavior if you have: `a > 10 && a is == 20`.  thanks~